packages/react-dom/src/__tests__/ReactDOMInput-test.js

```diff
@@ -797,16 +758,16 @@ describe('ReactDOMInput', () => {
       }
     }
 
-    const stub = ReactTestUtils.renderIntoDocument(<App />);
+    const stub = ReactDOM.render(<App />, container);
     const buttonNode = ReactDOM.findDOMNode(stub).childNodes[0];
     const firstRadioNode = ReactDOM.findDOMNode(stub).childNodes[1];
     expect(firstRadioNode.checked).toBe(false);
-    ReactTestUtils.Simulate.click(buttonNode);
+    dispatchEventOnNode(buttonNode, 'click');
     expect(firstRadioNode.checked).toBe(true);
   });
 
   it('should control radio buttons if the tree updates during render', () => {
```

Comment:
I’ve changed the order here slightly such that we can dispatch the event on the node that is already appended to the document.